src/utils.ts

```diff
@@ -0,0 +1,4 @@
+export function renderGitpodUrl(gitpodURL: string): string {
+    const baseURL = `${window.location.protocol}//${window.location.host}`;
+    return `${gitpodURL}#${baseURL}` + window.location.pathname;
```

Comment:
Haven't tested yet, but I'm a bit puzzled why there is no `/` between `${gitpodURL}` and `#`. (My current extension seems to already have a slash there.)

Reply:
Whey I type it by hand, I never use the `/`. Don't know if Firefox complains if there is no `/` or what the official standard is. However, I can add the `/` just to be sure.